cmd/provisioner-localpv/app/config.go

```diff
@@ -78,6 +78,33 @@ const (
 	//
 	KeyBDTag = "BlockDeviceTag"
 
+	//KeyNodeAffinityLabel defines the label key that should be
+	//used in the nodeAffinitySpec. Default is to use "kubernetes.io/hostname"
+	//
```

Comment:
Does the label `bd=bdc` as mentioned [here](https://github.com/openebs/openebs/issues/3094#issuecomment-739259419), gets added automatically to the node affinity spec along with the custom node affinity value?

Reply:
The `bd=bdc` is a different feature that needs to be implemented, this is only for supporting alternatives to `kubernetes.io/hostname` for node affinity.